spec/integration/direct_container_spec.rb

```diff
@@ -0,0 +1,151 @@
+require 'spec_helper'
+
+describe "Direct containers" do
+  describe "#directly_contains" do
+    context "when the class is ActiveFedora::File" do
+      before do
+        class FooHistory < ActiveFedora::Base
+           directly_contains :files, has_member_relation: ::RDF::URI.new("http://example.com/hasFiles")
+        end
+      end
+      after do
+        Object.send(:remove_const, :FooHistory)
+      end
+
+      let(:file) { o.files.build }
```

Comment:
I found it somewhat non-intuitive to use `build` rather than something line `add` or `create` but I guess since `build` is the ActiveRecord standard we should go with it. (sincerely, the the department of pointless comments)